tests/test_app/library/settings.py

```diff
@@ -17,6 +17,7 @@
 ALLOWED_HOSTS = ["*"]
 INSTALLED_APPS = [
     # Keep this above 'django.contrib.admin'
+    "fontawesomefree",
```

Comment:
if we want this to work, and for djangos static file finder to find the fontawesome files, then it will require everybody that has jazzmin installed to append this to their `INSTALLED_APPS` ..  this stops jazzmin being a one line drop in package for spicing up the admin